spec/controllers/openid_connect/logout_controller_spec.rb

```diff
@@ -552,6 +552,367 @@
         end
       end
     end
+
+    describe '#create' do
+      let(:id_token_hint) { valid_id_token_hint }
+      context 'when sending id_token_hint' do
+        subject(:action) do
+          post :create,
```

Comment:
is it possible to loop over the test cases but swap out the HTTP method so it doesn't need to be duplicated?

Reply:
Or maybe a [`shared_examples`](https://rspec.info/features/3-12/rspec-core/example-groups/shared-examples/)? (though I know `shared_examples` aren't universally held in high regard, I personally have no particular qualms of them)

Reply:
Yep!

Reply:
Fixed. I did the simplest version of this and shoved everything in the top-level describe statements for `#index` into `shared_examples`. I believe there is further room for improvement, but I didn't want this changeset to get too out of hand.